action.yml

```diff
@@ -82,16 +51,23 @@ runs:
     - name: Checkout
       uses: actions/checkout@v3
 
-    - name: Set vars
+    - name: config
+      uses: cloudposse/github-action-config-levels@nodejs20
```

Comment:
please set the name to be more descriptive. And point to a version if possible or comment why we cant

Reply:
Done